### DISCUSSION TITLE
Port to rust

### DISCUSSION THREAD

**Discussion by scriptmaster** [directus/directus#2859] [Feature Requests]
I love php, I don't deny its elegance.. however, some of the people I worked with, and most of the recruiters and directors look at php as some old language, under-estimate it, because of their own lack of knowledge and thus think they people are more powerful with their directing/recruiting/requirement knowledge with JVM GC 'tricks' and .Net Core, to server their enterprise clients and teams.

It would be like a good [revenj](https://github.com/ngs-doo/revenj) for these people to re-look into php, if this can be ported to rust and show them how difficult it is with their ideas and believes about under-estimating other people with strong php/nginx/lua background as well, especially with directus' great recent graphql support and vue.js why-not/why modernize to rust or zig is my request.  Kindly port.

**Comment by WoLfulus** [comment]
I'm not really sure what you're asking and/or requesting?

We do expect to see some ports of Directus API into other languages, that's one of the reasons why we are writing an API spec (https://github.com/directus/next/tree/main/packages/spec), but this will probably be a community effort atm. 

Porting to additional languages isn't our focus right now.

offtopic: rust is awesome and it's on my learning wishlist right now - together with Dutch haha. I'm not even sure which one has the biggest learning curve :P

**Reply by benhaynes** [reply]
Thanks, Wolf! 💯 

While we're not going to be creating any other official ports, I like the idea of community-lead initiatives!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
